.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
-**JIRA**: https://jira.appcelerator.org/browse/TIMOB-[TICKET-NUMBER]
+**JIRA:** https://jira.appcelerator.org/browse/[TICKET]
 
-**Additional description (optional)**: 
+Provide a clear PR title prefixed with `[TICKET]`
```

Comment:
That's already part of the guidelines, the PR template itself should be as template-like as possible.